web/index.php

```diff
@@ -43,8 +43,12 @@
     $container = new \Pimple\Container;
     $ezrpg = new Application($container);
 
-    // Get Config for the game
-    $ezrpg->getConfig();
+// Get configuration
+$config = $ezrpg->getConfig();
+
+echo $config['session']['lifetime'];
```

Comment:
Work in progress.  Should probably not be up as a PR, but is cool to use the diff as an example when discussing internally. Will be clean up before merged obviously :-)